src/scheduler/scheduler.ts

```diff
@@ -15,7 +15,7 @@ export enum SchedulerMode {
 
 export class SchedulerConfig {
     crontab: string = '* * * * *'; // Every minute
-    mode ?: SchedulerMode = SchedulerMode.ExactlyOncePerInterval;
+    mode ?: SchedulerMode = SchedulerMode.ExactlyOncePerIntervalWhenActive;
```

Comment:
I am tempted to say, rather than changing the default, eliminate the default.  Make people think about it... it has a pretty significant consequence.